### DISCUSSION TITLE
Windows - Is it possible to record loopback sound?

### DISCUSSION THREAD

**Discussion by alpvelioglu** [VladislavAntonyuk/MauiSamples#427] [Q&A] [answered]
Hi.

I want to record Windows sounds and display it like live captions.

**Comment by VladislavAntonyuk** [comment]
Hi, if you are talking about speech recognition, I haven’t seen such method to directly pass the audio stream. It receives the audio from your default microphone, so if you want to recognize the text from output stream, you can try making your dynamic louder.

**Comment by alpvelioglu** [comment]
Hi. Thanks for the answer.

I guess [Speech Recognition Engine](https://learn.microsoft.com/en-us/dotnet/api/system.speech.recognition.speechrecognitionengine.setinputtowavefile?view=dotnet-plat-ext-8.0&redirectedfrom=MSDN#System_Speech_Recognition_SpeechRecognitionEngine_SetInputToWaveFile_System_String_) transcript texts from recorded .wav file. If I record system sounds from Audacity, I can transcript it.

My main goal is, transcripting meetings and using the transcription I'll create summary from gpt-4. Thus, the audio from my microphone and Windows sound should be recorded in the same time.

**Comment by VladislavAntonyuk** [comment, marked as answer]
Maybe I didn't get the idea, but you can't use Windows output sound as input for your microphone. Assume you wear headphones.
What you can do is pass audio stream to recognition engine. But currently MCT speech recognition doesn't have the method for it.
